modules/assim.batch/tests/Rcheck_reference.log

```diff
@@ -87,37 +38,17 @@ use conditionally.
 * checking whether the namespace can be loaded with stated dependencies ... OK
 * checking whether the namespace can be unloaded cleanly ... OK
 * checking loading without being on the library search path ... OK
-* checking use of S3 registration ... OK
-* checking dependencies in R code ... NOTE
-Namespace in Imports field not imported from: ‘dplyr’
-  All declared Imports should be used.
+* checking dependencies in R code ... OK
 * checking S3 generic/method consistency ... OK
 * checking replacement functions ... OK
 * checking foreign function calls ... OK
 * checking R code for possible problems ... NOTE
-get.da.data: no visible binding for global variable ‘ensemble.samples’
-get.da.data : <anonymous>: no visible binding for global variable
-  ‘sa.samples’
-get.da.data: no visible binding for global variable ‘sa.samples’
 get.da.data : <anonymous>: no visible global function definition for
```

Comment:
Not required for the PR, but I'm curious what's up with `read.output.type` and why this one undefined variable was left?

Reply:
@mdietze I think I remember this. I was unable to find which package does read.output.type function belong to.

Reply:
Looking at those functions, I suspect `get.da.data.growth.R` and `get.da.data.R` can both be moved from the `/R` folder to the `/inst` folder. I don't have time to go through all the code in detail, but I think most of what's here is redundant with newer code by @istfer and others. This code doesn't appear to be operationally linked to other bits of the PDA and the value of retaining this code is primarily because it provides a record of what Carl Davidson did in his (still unpublished) Master thesis.